_unittest_solvers/test_26_emit.py

```diff
@@ -30,7 +30,8 @@ def aedtapp(add_app):
 
 
 @pytest.mark.skipif(is_linux, reason="Emit API fails on linux.")
-@pytest.mark.skipif(sys.version_info < (3,8) or sys.version_info >=(3, 12), reason="Emit API is only available for Python 3.8+.")
+@pytest.mark.skipif(sys.version_info < (3,10), reason="Emit API is only available for Python 3.10+.")
+@pytest.mark.skipif(sys.version_info > (3,12) and config["desktopVersion"] < "2025.1", reason="Emit API is only available for Python 3.12 in AEDT version 2025.1 and later.")
```

Comment:
```suggestion
@pytest.mark.skipif(sys.version_info >= (3,12) and config["desktopVersion"] < "2025.1", reason="Emit API is only available for Python 3.12 in AEDT version 2025.1 and later.")
```

Reply:
I see that this is incorrect for Python 3.12. I will make it `sys.version_info > (3,11)`. We can't use `>=` because `sys.version_info` has more than 2 elements so checking for equivalence with a 2-element tuple will always be false.